slc/component/matter-core-sdk/zigbee_cmp.slcc

```diff
@@ -0,0 +1,49 @@
+id: matter_zigbee_cmp
+package: matter
+label: zigbee cmp
+description: >
+  Enables Zigbee components required in a matter cmp project
+category: Matter|Core
+quality: production
+metadata:
+  sbom:
+    license: Zlib
+requires:
+# For compatibility with the Zigbee token manager
+  - name: common_token_manager_dynamic
+  - name: common_token_manager
+
+  - name: zigbee_address_table
+  - name: zigbee_application_bootloader
+  - name: zigbee_security_link_keys
+  - name: zigbee_source_route
+  - name: zigbee_stack_diagnostics
+  - name: zigbee_ipc
+  - name: zigbee_zll
+  - name: zigbee_zcl_framework_core
+  - name: zigbee_groups_server
+  - name: zigbee_identify
```

Comment:
I don't know if those are always used or if they are really clusters/device types dependent. 
It can stay here, and if we have issues with other device types ,we will revisit this

Reply:
Right, for now Identify is mandatory for both our supported apps, groups is only recommended for thermostat so I suppose we could split the component and level some of the components in the slcp for both apps.